gm4_desire_lines/data/desire_lines/functions/main.mcfunction

```diff
@@ -0,0 +1,4 @@
+scoreboard players add desire_lines gm4_random 27
+execute if score desire_lines gm4_random matches 100.. run scoreboard players remove desire_lines gm4_random 100
+
+execute as @a[gamemode=!spectator] at @s run function desire_lines:player
```

Comment:
Should also exclude `gamemode=creative` players.

Reply:
Should it? The 1.12 version didn't do that